src/onboarding/ChooseYourAdventure.tsx

```diff
@@ -61,7 +62,8 @@ function ChooseYourAdventure() {
       {
         text: t('chooseYourAdventure.options.add'),
         goToNextScreen: () => {
-          navigateHome({ params: { initialScreen: Screens.FiatExchange } })
+          navigateHome()
+          navigate(Screens.FiatExchangeCurrency, { flow: FiatExchangeFlow.CashIn })
```

Comment:
might be worth adding a code comment explaining why we navigate home and then immediately somewhere else